src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -38,6 +69,8 @@
     <GuidSymbol name="guidIssueVizPackageCmdSet" value="{FDEF405A-28C2-4AFD-A37B-49EF2B0D142E}">
       <IDSymbol name="SonarLintMenuGroup" value="0x1020" />
       <IDSymbol name="IssueVisualizationToolWindowCommandId" value="0x0100" />
+      <IDSymbol name="GotoNextLocationCommandId" value="4129" />
```

Comment:
It would be better if all of the `value` entries used the same base, rather than mixing decimal and hex.